compiler/plugins/target/ROCM/test/opt_pass_plugin/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+# Disabled on Windows until the shared library handling is cross-platform.
```

Comment:
Should we have a TODO/FIXME for this and link to a new tracking issue?

Reply:
The discussion on the linked PR is enough for me, unless this takes longer than a few days.